apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -23,7 +23,8 @@ export class UnconnectedTwoColumnActionBlock extends Component {
         url: PropTypes.string.isRequired,
         text: PropTypes.string.isRequired,
         target: PropTypes.string,
-        id: PropTypes.string
+        id: PropTypes.string,
+        color: PropTypes.oneOf(Object.values(Button.ButtonColor))
```

Comment:
this component is used in a bunch of places, but we aren't adding the `color` prop to most of them. do we want all of the following files to now use `Button.ButtonColor.brandSecondaryDefault` as the button color?
- LandingPage.jsx
- CertificateShare.jsx
- ProjectsPromo.jsx
- Incubator.jsx
- IncubatorBanner.jsx
- MarketingAnnouncementBanner.jsx
- SpecialAnnouncement.jsx

i don't know what many of these components do, but wanted to double-check that this color change (from gray to brandSecondaryDefault) in intended everywhere

Reply:
Yeah, true. Jessica said it's ok to update as much and as smooth as possible, so I think it's expected change